index.js

```diff
@@ -1,62 +1,113 @@
 'use strict';
 
 var postcss = require('postcss');
+var R = require('ramda');
 
 module.exports = postcss.plugin('postcss-responsive-type', function () {
 
   // Default parameters
-  var params = {
-    minSize: '12px',
-    maxSize: '21px',
-    minWidth: '420px',
-    maxWidth: '1280px'
+  var defaultParams = {
+    'font-size': {
+      minSize: '12px',
+      maxSize: '21px',
+      minWidth: '420px',
+      maxWidth: '1280px'
+    },
+    'line-height': {
+      minSize: '1rem',
+      maxSize: '2rem',
+      minWidth: '420px',
+      maxWidth: '1280px'
+    },
+    'letter-spacing': {
+      minSize: '1rem',
```

Comment:
These are some pretty wacky defaults. 1rem -> 2rem letter spacing would be completely illegible. Same for examples in the README.